README.md

```diff
@@ -88,6 +111,14 @@ func init() {
   info.Name, info.HTTPMethod, info.Path, info.Desc =
     "greets.list", "GET", "greetings", "List most recent greetings."
 
+  info = api.MethodByName("Add").Info()
+  info.Name, info.HTTPMethod, info.Path, info.Desc =
+    "greets.add", "PUT", "greetings", "Add a greeting."
+
+  info = api.MethodByName("Count").Info()
+  info.Name, info.HTTPMethod, info.Path, info.Desc =
+    "greets.count", "GET", "greetings/count", "Count all greetings."
+
```

Comment:
maybe also do the same thing here, define your neat `method()` closure? I liked it a lot.